WordPress/src/main/java/org/wordpress/android/ui/mysite/QuickStartRepository.kt

```diff
@@ -74,6 +95,45 @@ class QuickStartRepository
 
     fun setActiveTask(task: QuickStartTask) {
         activeTask.postValue(task)
+        val shortQuickStartMessage =
+                if (task == UPDATE_SITE_TITLE) {
+                    HtmlCompat.fromHtml(
+                            resourceProvider.getString(
+                                    R.string.quick_start_dialog_update_site_title_message_short,
+                                    SiteUtils.getSiteNameOrHomeURL(selectedSiteRepository.getSelectedSite())
+                            ), HtmlCompat.FROM_HTML_MODE_COMPACT
+                    )
+                } else {
+                    val activeTutorialPrompt = QuickStartMySitePrompts.getPromptDetailsForTask(task)
+                    quickStartUtils.stylizeQuickStartPrompt(
+                            activeTutorialPrompt!!.shortMessagePrompt,
+                            activeTutorialPrompt.iconId
+                    )
+                }
+        _onSnackbar.postValue(Event(SnackbarMessageHolder(UiStringText(shortQuickStartMessage))))
+    }
+
+    fun completeTask(task: QuickStartTask) {
+        selectedSiteRepository.getSelectedSite()?.let { site ->
+            // TODO Remove this before the feature is done
+            // Uncomment this code to mark a task as not completed for testing purposes
+//            if (quickStartStore.hasDoneTask(site.id.toLong(), task)) {
+//                quickStartStore.setDoneTask(site.id.toLong(), task, false)
+//                refresh.value = false
+//                return
+//            }
+            if (task != activeTask.value) return
+            activeTask.value = null
+            if (quickStartStore.hasDoneTask(site.id.toLong(), task)) return
+            // If we want notice and reminders, we should call QuickStartUtils.completeTaskAndRemindNextOne here
+            quickStartStore.setDoneTask(site.id.toLong(), task, true)
```

Comment:
Judging by the code in `QuickStartUtils`, I'm guessing we're missing a call to `AnalyticsTracker` here in order to track the completed task, what do you think?

Reply:
I've added tracking in https://github.com/wordpress-mobile/WordPress-Android/pull/13828/commits/3d5e3975028c63fc455aeafce0e32354a69b8a2f